cips/cip-template.md

```diff
@@ -14,13 +14,13 @@ requires: CIP number(s). Only required when you reference an CIP in the `Specifi
 **READ CIP-1 BEFORE USING THIS TEMPLATE!**
 This is the suggested template for new CIPs. After you have filled in the requisite fields, please delete these comments. Note that an CIP number will be assigned by an editor. When opening a pull request to submit your CIP, please use an abbreviated title in the filename, `cip-draft_title_abbrev.md`. The title should be 44 characters or less. It should not repeat the CIP number in title, irrespective of the category.
 
-**TODO: Remove the note before submitting**
```

Comment:
[not blocking] this is the only thing that caught my eye @rootulp. should these remain bold? or a rule be ignored?

Reply:
yep it's MD036 https://github.com/markdownlint/markdownlint/blob/main/docs/RULES.md#md036---emphasis-used-instead-of-a-header

Reply:
Sync'ed with Josh offline and we agreed to remove the bold and keep the rule on.